src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.html

```diff
@@ -311,11 +311,10 @@
              let-column="data.column"
              let-value="data.value">
   <span *ngFor="let item of (value | array); last as last">
```

Comment:
```suggestion
  <cds-tag *ngFor="let item of (value | array); last as last">
```
Should this be changed as well?